chain-extensions/types/xvm/src/lib.rs

```diff
@@ -41,8 +41,7 @@ impl From<CallError> for XvmExecutionResult {
             SameVmCallNotAllowed => 2,
             InvalidTarget => 3,
             InputTooLarge => 4,
-            BadOrigin => 5,
-            ExecutionFailed(_) => 6,
+            ExecutionFailed(_) => 5,
         };
         Self::Err(error_code)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ExecutionFailed` variant has been modified from being part of a `match` block to being standalone. This likely impacts the handling of error codes. Check whether this change aligns with the overall error handling strategy of the module.
```diff
             ExecutionFailed(_) => 5,
         };
+        Self::Err(error_code)
     }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->